netctl/netctl.go

```diff
@@ -726,3 +738,67 @@ func listAppProfEpgs(ctx *cli.Context) {
 		os.Stdout.WriteString(groups)
 	}
 }
+
+func listExternalContracts(ctx *cli.Context) {
+	argCheck(0, ctx)
+
+	extContractsGroups, err := getClient(ctx).ExtContractsGroupList()
+	errCheck(ctx, err)
+
+	if ctx.Bool("json") {
+		dumpJSONList(ctx, extContractsGroups)
+	} else if ctx.Bool("quiet") {
+		contractsGroupNames := ""
+		for _, extContractsGroup := range *extContractsGroups {
```

Comment:
Why do you need to use *extContractsGroups here? I think range over extContractGroups should be enogh.